src/GLib/gtype.jl

```diff
@@ -18,22 +18,22 @@ end
 const fundamental_types = (
     #(:name,      Ctype,            JuliaType,      g_value_fn)
     (:invalid,    Void,             Void,           :error),
-    (:void,       Nothing,          Nothing,        :error),
+    (:void,       Void,          Void,        :error),
     (:GInterface, Ptr{Void},        GInterface,     :error),
     (:gchar,      Int8,             Int8,           :schar),
-    (:guchar,     Uint8,            Uint8,          :uchar),
+    (:guchar,     UInt8,            UInt8,          :uchar),
     (:gboolean,   Cint,             Bool,           :boolean),
-    (:gint,       Cint,             None,           :int),
-    (:guint,      Cuint,            None,           :uint),
-    (:glong,      Clong,            None,           :long),
-    (:gulong,     Culong,           None,           :ulong),
+    (:gint,       Cint,             Union(),           :int),
+    (:guint,      Cuint,            Union(),           :uint),
+    (:glong,      Clong,            Union(),           :long),
+    (:gulong,     Culong,           Union(),           :ulong),
```

Comment:
I used `Union()` because initially I was striving for julia 0.3 compatibility. But I can change that.

> Union() is not giving deprecation warning yet. Should it?

Probably worth filing a julia issue about it.